trace.go

```diff
@@ -137,21 +152,11 @@ func (e *traceExporter) flush() {
 	e.payload.reset()
 }
 
-// stop cleanly stops the exporter, flushing any remaining spans to the transport and
-// reporting any errors. Make sure to always call Stop at the end of your program in
-// order to not lose any tracing data. Only call Stop once per exporter. Repeated calls
-// will cause panic.
+// stop() notify loop() via exit channel to cleanly stops the exporter.
+// Make sure to always call stop() at the end of your program in order to
```

Comment:
Delete the remaining comment lines. That part is already explained in a different spot: https://github.com/DataDog/opencensus-go-exporter-datadog/blob/master/datadog.go#L42-L45